resurfemg/helper_functions.py

```diff
@@ -1372,15 +1375,16 @@ def find_peak_in_breath(
     :type start_index: int
     :param end_index: which index number the breath ends on
     :type end_index: int
-    :param smooth_algorithm: algorithm for smoothing (none or 'mid-savgol')
+    :param smooth_algorithm: algorithm for smoothing (none or
+    'mid-savgol' or 'convy')
     :type smooth_algorithm: str
 
-    :returns: index of max point, value at max point
+    :returns: index of max point, value at max point, smoothed value
     :rtype: tuple
     """
     new_array = array[start_index: (end_index+1)]
     if smooth_algorithm == 'mid_savgol':
-        new_array = savgol_filter(
+        new_array2 = savgol_filter(
             new_array, int(len(new_array)),
```

Comment:
```suggestion
            abs(new_array), int(len(new_array)),
```

I guess that also here we want to use the absolute values of the signal